src/components/Home/Main/styles.css

```diff
@@ -0,0 +1,251 @@
+    -webkit-background-clip: text;
+    -webkit-text-fill-color: transparent;
+}
+.top-section-info>h3{
+    font-size: 2.5rem;
+    background: -webkit-linear-gradient(45deg, rgb(80, 80, 80) 0%, rgb(170, 170, 170) 30%, rgba(255,254,255,1) 100%);
+    -webkit-background-clip: text;
+    -webkit-text-fill-color: transparent;
+    margin-bottom: 1rem;
+}
+.top-section-info>button{
+    font-size: 2rem;
+    width: 250px;
+    padding: 0.5rem 0;
+    border: none;
+    border-radius: 10px;
+    transition: 0.5s;
+    margin-bottom: 2rem;
+}
+.top-section-info>button:hover{
+    background: var(--primary-clr-hover);
+    color: var(--primary-clr);
+}
+
+.top-section-image{
+    height: 400px;
+    width: 400px;
+    border-radius: 15px;
+    margin-right: 3rem;
+}
+
+.fa.fa-angle-left{
+    display: inline-block;
+    position: absolute;
+    left: 10px;
+    font-size: 2.5rem;
+    color: var(--primary-clr);
+}
+.fa.fa-angle-right{
+    display: inline-block;
+    position: absolute;
+    right: 10px;
+    font-size: 2.5rem;
+    color: var(--primary-clr);
+}
+
+.fa.fa-angle-left:hover, .fa.fa-angle-right:hover{
+    color: var(--primary-clr-hover);
+}
+
+
+
+/* featured-section */
+.main-container.featured-section{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    gap: 2rem;
+    min-width: 100vw;
+    background-image: linear-gradient(to right, #505050, #bebebe);
+    padding: 3rem 2rem;
+}
+
+.main-container.featured-section>h1{
+    background: -webkit-linear-gradient(45deg, rgb(0, 0, 0) 0%, rgb(50, 50, 50) 50%, rgb(90, 90, 90) 100%);
+    -webkit-background-clip: text;
+    -webkit-text-fill-color: transparent;
+    font-size: 4rem;
+}
+
+.feature-section-info{
+    display: flex;
+    position: relative;
+    gap: 2rem;
+    flex-wrap: wrap;
+    justify-content: center;
+}
+.feature-list{
+    list-style: none;
+    display: flex;
+    position: relative;
+}
+.feature-img{
+    width: 250px;
+    overflow: hidden;
+}
+.feature-badge{
+    font-size: 1.5rem;
+    width: 100%;
+    display: inline-block;
+    text-align: center;
+    position: absolute;
+    border-radius: 10px;
+    bottom: 0;
+    background-color: rgba(255, 255, 255, 0.5);
+    color: black;
+}
+
+
+/* cards-section */
+.card-section{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    gap: 2rem;
+    width: 100vw;
+    margin-top: 2rem;
+    flex-wrap: wrap;
+    margin-bottom: 2rem;
+}
+
+.card-wrapper{
+    display: flex;
+    gap: 3rem;
+    align-items: center;
+    justify-content: center;
+    flex-wrap: wrap;
+}
+
+.card-section>h1{
+    background: -webkit-linear-gradient(45deg, rgb(255, 255, 255) 0%, rgb(200, 200, 200) 50%, rgb(110, 110, 110) 100%);
+    -webkit-background-clip: text;
+    -webkit-text-fill-color: transparent;
+    font-size: 4rem;
+}
+
+.card {
```

Comment:
Try using your component library if possible that way, you will also be able to create a project with component library which is needed in the end.